TYPO3.Flow/Classes/TYPO3/Flow/Resource/Publishing/MessageCollector.php

```diff
@@ -0,0 +1,106 @@
+<?php
+namespace TYPO3\Flow\Resource\Publishing;
+
+/*
+ * This file is part of the TYPO3.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Error\Error;
+use TYPO3\Flow\Error\Message;
+use TYPO3\Flow\Error\Notice;
+use TYPO3\Flow\Error\Warning;
+use TYPO3\Flow\Exception;
+use TYPO3\Flow\Log\SystemLoggerInterface;
+
+/**
+ * Notification Collector
+ *
+ * @Flow\Scope("singleton")
+ */
+class MessageCollector
+{
+    /**
+     * @var \SplObjectStorage
+     */
+    protected $backends;
```

Comment:
`$backends` is a weird name. This contains messages, so why not name it `$messages`?